drivers/gles3/storage/particles_storage.h

```diff
@@ -154,6 +156,7 @@ class ParticlesStorage : public RendererParticlesStorage {
 		double inactive_time = 0.0;
 		bool emitting = false;
 		bool one_shot = false;
+		real_t amount_ratio = 1.0;
```

Comment:
```suggestion
		float amount_ratio = 1.0;
```
This can be a float since the setter and the shader both use floats